regression_test_script

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+set -e
+echo "Starting up ..."
+
+default_time_out=30
+default_image_name="opendap/hyrax:snapshot"
+default_endpoint="http://localhost:8080/opendap/"
+
+time_out="${default_time_out}"
+test_image_name="${default_image_name}"
+end_point="${default_endpoint}"
+netrc_fileurl=""
+
+while getopts "ht:i:e:n:" OPTION; do
+    case $OPTION in
+    h)
+        echo "-h help -t time_out -i image_name -e end_point_url"
+	exit 0
+        ;;
+    t)
+        time_out=$OPTARG
+        ;;
+    i)
+        test_image_name=$OPTARG
+        ;;
+    e)
+        end_point=$OPTARG
+        ;;
+    n)
+        netrc_fileurl=$OPTARG
```

Comment:
Doing it like this:
```
netrc_file="--netrc ${$OPTARG}"
```
Means that it can be included in the call to testsuite without a conditional.